File: 프로그래머스/0/120891. 369게임/369게임.java
class Solution {
    public int solution(int order) {
        int answer = 0;
        String s = String.valueOf(order);
        
        for (int i = 0; i < s.length(); i++ ){
            char cs = s.charAt(i);
            if (cs == '3' || cs == '6' || cs == '9'){
                answer ++ ;
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/120891. 369게임/369게임.kt
class Solution {
    fun solution(order: Int): Int {
        var answer: Int = 0
        val orderStr = order.toString()
        
        for (i in 0 until orderStr.length){
            if (orderStr[i] == '3' || orderStr[i] == '6' || orderStr[i] == '9')
            answer++
        }
        return answer
    }
}